components/Page.tsx

```diff
@@ -1,11 +1,9 @@
 import { storyblokEditable, StoryblokComponent } from '@storyblok/react';
-
 const Page = ({ blok }) => (
   <main className='mt-4 text-center' {...storyblokEditable(blok)}>
-    {blok.body.map((nestedBlok) => (
+    {Array.isArray(blok.body) && blok.body.map((nestedBlok) => (
```

Comment:
@Nancy-Kaur11 I think defaulting body = [] to empty array is simpler that isArray check. Since array would be empty, it won't render anything.